main/Badge/README.md

```diff
@@ -0,0 +1,176 @@
+# Badge  
+> [Badge] component that can be used inside product. Has `OpacityVisible` state, `badgeZero` state.  
+<br/>
+<br/>
+
+## Preview 
+
+| Badge(default) | Badge(color, count, showZero) |
+| :---------------: | :--------------: |
+| <img src="https://user-images.githubusercontent.com/37579661/92614370-eff66380-f2f6-11ea-9f46-35f5cc871a72.png" width= "50%"/> | <img src="https://user-images.githubusercontent.com/37579661/92614395-f553ae00-f2f6-11ea-9663-f507d62c0a7c.png" width= "50%"/> |
+
+| Badge(maximumCount, opacityVisible) | Badge(border, textColor) | Badge(variant, position) |
+| :---------------: | :--------------: | :-------------: |
+| <img src="https://user-images.githubusercontent.com/37579661/92614433-01d80680-f2f7-11ea-99a5-ed89ff14216c.png" width= "100%"/> | <img src="https://user-images.githubusercontent.com/37579661/92614441-056b8d80-f2f7-11ea-9e40-0ca17b6db965.png" width= "100%"/> |  <img src="https://user-images.githubusercontent.com/37579661/92616226-ee2d9f80-f2f8-11ea-9573-7ebec6f36ce6.png" width= "100%"/> |
+
+
+
+## Props  
+  
+|                       | required | types                 | default   |description |
+| :-------------------: | -------- | :-------------------: | :-------: | ---------- |
+| **count**             |          | number                |    1      | Proportionally controls opacity of badge color |
+| **color**             |          | string                | '34AFF9'  | Set badge color |
+| **maximumValue**      |          | number                |     300   | Set maximum limit where opacity stay 1.0, and if count exceeds maximumValue, badge color opacity sets to 0.6 | 
+| **showZero**          |          | boolean               |           | Set whether to show '0' count number in badge or not |
```

Comment:
Does not `showZero` have a default value?

Reply:
> Does not `showZero` have a default value?

working on it, modificiations will be merged soon . thank you!